triton-client/src/components/DatasetCard.tsx

```diff
@@ -73,34 +96,40 @@ function DatasetCard({ datasetID }: DatasetCardProps) {
 	const requestDetails = useMemo(() => {
 		return SUPPORTED_DOWNLOAD_TYPES.map((type) => {
 			const req = requestByType[type]
-			if (req) {
+			if (req && !!!req.should_delete) {
```

Comment:
Hm!!!?

Reply:
You only need one `!`.

Reply:
should_delete is an int ... this was to indicate a type cast to boolean.

Reply:
One `!` is enough to turn anything into a boolean ;)

Reply:
in the world of typescript it is clearly meaningless but it was to make it obvious it was not a native bool and it would get converted. i'll clean it.